marimo/_plugins/stateless/test_image.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Marimo. All rights reserved.
+from marimo._plugins.stateless.image import image
+from marimo._runtime.conftest import ExecReqProvider
+from marimo._runtime.context import get_context
+from marimo._runtime.runtime import Kernel
+
+
+def test_image() -> None:
+    result = image(
+        "https://marimo.io/logo.png",
+    )
+    assert result.text == "<img src='https://marimo.io/logo.png' />"
+
+
+def test_image_bytes_io(k: Kernel, exec_req: ExecReqProvider) -> None:
+    k.run(
+        [
+            exec_req.get(
+                """
+                import io
+                import marimo as mo
+                bytestream = io.BytesIO(b"\x89PNG\r\n\x1a\n\x00\x00\x00")
```

Comment:
I tried testing locally on Ubuntu, and Linux doesn't like this bytestring, saying that it contains `null` bytes (which causes the test to fail). In particular cell registration fails with

```
(UnknownError(msg='Traceback (most recent call last):
File "/home/akshay/src/marimo-team/marimo/marimo/_runti...compile(source, filename, mode, flags,
ValueError: source code string cannot contain null bytes\n', type='unknown'),
```

Reply:
The same error affects `test_image_bytes`

Reply:
More context: https://github.com/python/cpython/issues/96670#issuecomment-1249054660

So it seems like we can't hardcode the bytestring as a string literal. Instead you could download the image and convert it to bytes at runtime?

Reply:
i just updated with `b"hello"`. it doesnt need to be an actual image